ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -68,6 +68,11 @@ def clip_by_norm(t, clip_norm, axes=None):
     return t_clip
 
 
+@to_ivy_arrays_and_back
+def cond(pred, true_fn=None, false_fn=None, name=None):
+    return ivy.if_else(pred, true_fn, false_fn, ivy.array([]))
```

Comment:
Ivy functional API doesn't have this function implemented you'll need to implement a composition for it